tests/dryrun_test.go

```diff
@@ -67,9 +64,9 @@ var _ = Describe("[sig-compute]Dry-Run requests", decorators.SigCompute, func()
 		var vmi *v1.VirtualMachineInstance
 		resource := "virtualmachineinstances"
 
-		BeforeEach(func() {
-			vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
-		})
+		vmi = libvmi.NewAlpine()
+		vmi.ObjectMeta.Namespace = testsuite.GetTestNamespace(vmi)
```

Comment:
`vmi.ObjectMeta.Namespace = testsuite.GetTestNamespace(vmi)` should not be required.